templates/etc/systemd/system/gitlab-pages.service.j2

```diff
@@ -8,13 +8,13 @@ PartOf=gitlab.slice
 
 [Service]
 Slice=gitlab.slice
-Type=forking
+Type=simple
 User={{ gitlab_user }}
 WorkingDirectory={{ gitlab_pages_path }}
 SyslogIdentifier=gitlab-pages
 PIDFile={{ gitlab_app_root_path }}/gitlab/tmp/pids/gitlab-pages.pid
 
-ExecStart={{ gitlab_pages_dir }}/gitlab-pages -pages-domain {{ gitlab_pages_domain }} -pages-root {{ gitlab_pages_path }} -listen-proxy 127.0.0.1:{{ gitlab_pages_port }}
+ExecStart={{ gitlab_app_root_path }}/gitlab-pages/gitlab-pages -pages-domain {{ gitlab_pages_domain }} -pages-root {{ gitlab_pages_path }} -listen-proxy 127.0.0.1:{{ gitlab_pages_port }}
```

Comment:
Sure, might be a good idea to change that when variables are moved from `vars/main.yml` to `defaults/main.yml` to keep them synchronized.